### PR TITLE
Add Python 3.5 test support

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

**Comment by IlyaSkriblovsky** [comment]
Looks great

**Comment by psi29a** [comment]
Thanks! :)

**Comment by psi29a** [comment]
Can you also add it here?
https://github.com/twisted/txmongo/blob/master/setup.py#L28

File: README.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ protocol on non-blocking sockets. The API derives from the original PyMongo.
 
 Compatibility
 -------------
-Python 2.7, 3.0, 3.3, 3.4 and PyPy.
+Python 2.7, 3.0, 3.3, 3.4, 3.5 and PyPy.
```
Comment (IlyaSkriblovsky) [inline README.rst]: I believe we can safely drop 3.0 and leave only "*Python 2.7, 3.3+ and PyPy*" here. Nobody seems to support Python 3.0–3.2, including Twisted.

**Comment by touilleMan** [comment]
PR updated

**Comment by psi29a** [comment]
:+1:

**Pull request merged**